backend/templates/registration/invite_signup_email.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}{% autoescape off %}
+{% blocktranslate %}You've been invited to join Pokebattle!{% endblocktranslate %}
+
+{% translate "Please go to the following page and register:" %}
+{% block reset_link %}
+{{ protocol }}://{{ '127.0.0.1:8000' }}{% url 'signup_confirm' uidb64=uid token=token %}
```

Comment:
{{ '127.0.0.1:8000' }} -> `127.0.0.1:8000`

It's the same thing. As it is now you're evaluating a string.